src/openvtb/components/verzoeken/models.py

```diff
@@ -301,11 +295,13 @@ def __str__(self):
 
     def clean(self):
         super().clean()
-        if not self.verzoek_type.last_version:
+        if self.version not in self.verzoek_type.versions.values_list(
+            "version", flat=True
+        ):
```

Comment:
I think using `.exists` is slightly more efficient here, because in that case the database stops executing the query as soon as it finds a matching row, and the resulting data that is transferred is just a boolean
```suggestion
        if not self.verzoek_type.versions.filter(version=self.version).exists()
```